src/st/no0/no0.h

```diff
@@ -31,9 +31,13 @@ typedef enum EntityIDs {
     /* 0x14 */ E_ID_14,
     /* 0x15 */ E_GREY_PUFF,
     /* 0x20 */ E_CLOCK_ROOM_SHADOW = 0x20,
-    /* 0x21 */ E_ID_21,
-    /* 0x24 */ E_DIPLOCEPHALUS_TAIL = 0x24,
+    /* 0x21 */ E_DIPLOCEPHALUS,
+    /* 0x22 */ E_ID_22,
+    /* 0x23 */ E_ID_23,
+    /* 0x24 */ E_DIPLOCEPHALUS_TAIL,
     /* 0x25 */ E_DIPLOCEPHALUS_FIREBALL,
+    /* 0x26 */ E_DIPLOCEPHALUS_TORSO,
+    /* 0x27 */ E_OLROX_DROOL,
```

Comment:
This entity should probably be renamed something more generic at some point as it's not Olrox related